tests/automation/input_validations.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { englishStrippedStr } from '../locale/localizedString';
+import { sessionTestOneWindow } from './setup/sessionTest';
+import {
+  clickOnTestIdWithText,
+  grabTextFromElement,
+  typeIntoInput,
+  waitForTestIdWithText,
+} from './utilities/utils';
+
+sessionTestOneWindow('Onboarding incorrect seed', async ([window]) => {
+  const incorrectSeed =
+    'ruby bakery illness push rift reef nabbing bawled hope zork silk lobster hope';
+  const expectedError = englishStrippedStr(
+    'recoveryPasswordErrorMessageIncorrect',
+  ).toString();
+  await clickOnTestIdWithText(window, 'existing-account-button');
+  await typeIntoInput(window, 'recovery-phrase-input', incorrectSeed);
+  await clickOnTestIdWithText(window, 'continue-button');
+  await waitForTestIdWithText(window, 'session-error-message');
+  const actualError = await grabTextFromElement(
+    window,
+    'data-testid',
+    'session-error-message',
+  );
+  if (actualError !== expectedError) {
+    throw new Error(
+      `Expected error message: ${expectedError}, but got: ${actualError}`,
+    );
+  }
+});
+
+sessionTestOneWindow('Onboarding too short seed', async ([window]) => {
+  const incorrectSeed = 'zork';
+  const expectedError = englishStrippedStr(
+    'recoveryPasswordErrorMessageShort',
+  ).toString();
+  await clickOnTestIdWithText(window, 'existing-account-button');
+  await typeIntoInput(window, 'recovery-phrase-input', incorrectSeed);
+  await clickOnTestIdWithText(window, 'continue-button');
+  await waitForTestIdWithText(window, 'session-error-message');
+  const actualError = await grabTextFromElement(
+    window,
+    'data-testid',
+    'session-error-message',
+  );
+  if (actualError !== expectedError) {
+    throw new Error(
+      `Expected error message: ${expectedError}, but got: ${actualError}`,
+    );
+  }
+});
+
+sessionTestOneWindow('Onboarding wrong seed', async ([window]) => {
+  const incorrectSeed =
+    'ruby bakery illness push rift reef nabbing bawled hope ruby silk lobster hope ruby ruby ruby';
+  const expectedError = englishStrippedStr(
+    'recoveryPasswordErrorMessageGeneric',
+  ).toString();
+  await clickOnTestIdWithText(window, 'existing-account-button');
+  await typeIntoInput(window, 'recovery-phrase-input', incorrectSeed);
+  await clickOnTestIdWithText(window, 'continue-button');
+  await waitForTestIdWithText(window, 'session-error-message');
+  const actualError = await grabTextFromElement(
+    window,
+    'data-testid',
+    'session-error-message',
+  );
+  if (actualError !== expectedError) {
+    throw new Error(
+      `Expected error message: ${expectedError}, but got: ${actualError}`,
+    );
+  }
+});
```

Comment:
![image](https://github.com/user-attachments/assets/f09909ff-d61a-4dc3-8827-1b17c354ea11)
those 3 are massively the same. Maybe create a test that takes a seed and an expected error string, and make that function called three times with each set of options?